rich/console.py

```diff
@@ -2373,6 +2377,18 @@ def stringify(value: object) -> str:
             if clear:
                 self._record_buffer.clear()
 
+        if unique_id is not None:
+            import warnings
+
+            msg = (
+                "'unique_id' parameter passed to SVG export.  This has the potential "
+                "to cause namespace collisions if the SVG is embedded in a document."
+            )
+            warnings.warn(msg)
+
+        if encapsulated is False:
+            unique_id = "terminal"
```

Comment:
Hmm.. this probably has the potential to cause as-many, if not more namespace collisions than (most?) user-selected `unique_id` parameters.

Reply:
(the idea is that `encapsulated=False` would only be set when the caller knows they intend to place the SVG in a context where collisions shouldn't occur.  is that a reasonable expectation, though?)

Reply:
I don't think that the original caller is the problem, to be honest - if anyone discovers this parameter and provides a specific value, then _they_ probably knew what they were doing.

The problem is that the document that is emitted may end up being re-used and embedded in other contexts that the caller themselves may not be aware of.

To that extent, encapsulation should really be near-required for formats that _don't_ encapsulate by default.  So we should raise the same warning here -- unless the output format is _known_ to be self-encapsulating.